### PR TITLE
GEODE-5601: Serialize running of acceptance tests in lieu of containerizing them

### PR DESCRIPTION
**Pull request by pdxrunner**
GEODE-5601: Serialize running of acceptance tests in lieu of containerizing them

Fix flaky AccpetanceTest runs by running them serially. This does incur a runtime penalty
(< 10%) for this category of test.

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [X] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [X] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [X] Is your initial contribution a single, squashed commit?

- [X] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by pdxrunner** [comment]
This should be ready to go now. One failure precheckin failure in DistributedTest (still flaky: org.apache.geode.cache30.DistributedNoAckRegionCCEDUnitTest > testTXRmtMirror)

**Comment by upthewaterspout** [comment]
Looks good, but I'm curious - since you added a new PARALLEL_GRADLE flag, do we really need a whole new execute-acceptance-test script?

**Comment by pdxrunner** [comment]
No. The execute-acceptance-test.sh script was used in the first commit, but is not needed in the final subsequent commit. It was an oversight to leave it.

File: ci/pipelines/geode-build/jinja.template.yml [hunk 1]
```diff
@@ -274,7 +274,11 @@ jobs:
           GRADLE_TASK_OPTIONS: ""
           {{ common_test_params(test) | indent(10) }}
         run:
+          {%- if test.name=="Acceptance" %}
+          path: geode/ci/scripts/execute_acceptance_tests.sh
```
Comment (upthewaterspout) [inline ci/pipelines/geode-build/jinja.template.yml]: ^^^execute_acceptance_test is still used here in the template.

File: ci/pipelines/shared/jinja.variables.yml [hunk 2]
```diff
@@ -30,10 +30,11 @@ tests:
...
   ARTIFACT_SLUG: acceptancetestfiles
   CALL_STACK_TIMEOUT: "1800"
-  DUNIT_PARALLEL_FORKS: "7"
-  GRADLE_TASK: :geode-assembly:acceptanceTest geode-connectors:acceptanceTest
+  DUNIT_PARALLEL_FORKS: "0"
+  GRADLE_TASK: acceptanceTest
   execute_test_timeout: 45m
-  PARALLEL_DUNIT: "true"
+  PARALLEL_DUNIT: "false"
+  PARALLEL_GRADLE: "false"
```
Comment (pivotal-jbarrett) [inline ci/pipelines/shared/jinja.variables.yml]: This still needs to get tied into the tasks in the templates.

**Pull request merged**